cmd/kubeadm/app/cmd/reset.go

```diff
@@ -44,7 +44,7 @@ var (
 		The reset process does not reset or clean up iptables rules or IPVS tables.
 		If you wish to reset iptables, you must do so manually.
 		For example:
-		iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+		iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X && iptables -t nat -X && iptables -t mangle -X
```

Comment:
@kubernetes/sig-cluster-lifecycle-pr-reviews 
PTAL

Reply:
I have some potential concerns that we may be telling users to kill rules that the user may be relying on, especially on local system installs, but that already exists with the existing command we are telling them to run.

Reply:
i think we might want to remove the example and leave it to the users to figure things out by reading the docs for `iptables`.

proposing:
> If you wish to reset iptables, you must do so manually by using the `iptables` command.

Reply:
> we might want to remove the example and leave it to the users to figure things out by reading the docs for iptables

Agree +1000!

Reply:
@tnqn would you mind making the proposed change in https://github.com/kubernetes/kubernetes/pull/79648#discussion_r300121457
as part of this PR?

the idea is to more the iptables example completely and have that sentence instead.

Reply:
Sure, done. Thanks for the suggestion.